data/helpers/conversations/spamScore.ts

```diff
@@ -80,7 +81,10 @@ export const computeConversationsSpamScores = async (
     const senderSpamScore = sendersSpamScores[senderKey];
     if (
       !conversation.messagesIds.length &&
-      typeof senderSpamScore === "number"
+      typeof senderSpamScore === "number" &&
+      // 1:1 Conversations without messages and no specific sender spam score
+      // should not get a spam score now (waiting for first message)
+      (conversation.isGroup || senderSpamScore !== 0)
```

Comment:
Hey @nmalzieu - I'm new here so likely lacking context, but the 1:1 in the comments doesn't seem to line up with the check of `.isGroup`. I would imagine that 1:1 conversations would **not** be a group, correct?

Could you help me understand what I'm missing there?

Reply:
This will leave the conversation spam score null or undefined, which will be treated as spam:

https://github.com/ephemeraHQ/converse-app/pull/815/files#diff-7eda797ad01ac0fefcc17a9a00c4f7fb9607e883f5a65321163a75e96ec1c889R808

Reply:
Yes exactly, a 1:1 conversation will be in hidden until we have a first message in it. Most clients create convo then send message almost immediatly, but if we compute spam score too early then we never recompute it.
Also I guess empty 1:1 convos we don't really care about so it's ok if they're in hidden requests?

Reply:
Great. We sometimes see artificially empty 1:1 conversations in the main inbox during initial load/sync. Holding them in hidden requests until they have a message and have been scored makes sense for now.

Reply:
Convos I have approved (protocol consent) will still appear in inbox even if no message (or during initial sync).